person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  # Constructor
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
Good work adding the getters and setters. 

- [ ] To follow best practices, kindly move these two plain methods to the beginning of the class. Usually, for best reading code, plain methods are written at the beginning of the class.